### PR TITLE
Refs EN-5887

### PR DESCRIPTION
**Pull request by rozap**
* Use the old idle-timeout setting

### CODE REVIEW

File: socrata-http-jetty/src/main/scala/com/socrata/http/server/AbstractSocrataServerJetty.scala [hunk 1]
```diff
@@ -310,7 +306,7 @@ object AbstractSocrataServerJetty {
     extraHandlers: List[Handler => Handler] = Nil,
     errorHandler: Option[HttpRequest => HttpResponse] = None,
     poolOptions: Pool.Options = Pool.defaultOptions,
-    idleTimeout: Int = 30000
+    idleTimeoutMs: Int = 30000
```
Comment (rjmac) [inline socrata-http-jetty/src/main/scala/com/socrata/http/server/AbstractSocrataServerJetty.scala]: Parameter names are part of the public API...

Comment (rjmac) [inline socrata-http-jetty/src/main/scala/com/socrata/http/server/AbstractSocrataServerJetty.scala]: oh nm, this isn't a public function.

File: version.sbt [hunk 2]
```diff
@@ -1 +1 @@
-version in ThisBuild := "3.9.3-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "3.10.3-SNAPSHOT"
```
Comment (michaelb990) [inline version.sbt]: why?

File: socrata-http-jetty/src/main/scala/com/socrata/http/server/AbstractSocrataServerJetty.scala [hunk 3]
```diff
@@ -325,7 +321,6 @@ object AbstractSocrataServerJetty {
     override def withExtraHandlers(h: List[Handler => Handler]) = copy(extraHandlers = h)
     override def withErrorHandler(h: Option[HttpRequest => HttpResponse]) = copy(errorHandler = h)
     override def withPoolOptions(poolOpt: Pool.Options) = copy(poolOptions = poolOpt)
-    override def withIdleTimeout(it: Int) = copy(idleTimeout = it)
```
Comment (michaelb990) [inline socrata-http-jetty/src/main/scala/com/socrata/http/server/AbstractSocrataServerJetty.scala]: this does seem like it is part of the public API though, are we worried about this breaking things?

Comment (rjmac) [inline socrata-http-jetty/src/main/scala/com/socrata/http/server/AbstractSocrataServerJetty.scala]: Yes, that definitely is.  Better to `deprecated` it and return `this` than to remove it completel.y

**Pull request merged**